src/utils.rs

```diff
@@ -0,0 +1,60 @@
+//File for utility functions used local to the system, such as command exeution
```

Comment:
I'm find with the naming for now, but `utils` might be a little generic and we may find a way to refactor it in the future. Perhaps combining this module with creating shell completions code and calling the module `shell` ? 

Definitely merge as is and we can let it evolve from here.

Reply:
Renamed file; will update function slater.